crates/tags/src/entry.rs

```diff
@@ -1,42 +1,42 @@
 // SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
 // SPDX-License-Identifier: Apache-2.0
 
+use io::Error;
+use std::io;
 use std::ops::{Deref, DerefMut};
 use std::str::FromStr;
 
-use drawbridge_hash::Error;
 use drawbridge_http::http::{Request, StatusCode};
 use drawbridge_http::{async_trait, FromRequest};
+use serde::{Deserialize, Serialize};
 
-use serde::Serialize;
+#[derive(Clone, Deserialize, Serialize)]
+pub struct Entry(Vec<u8>);
 
-#[derive(Clone, Serialize)]
-pub struct Hash(drawbridge_hash::Hash);
-
-impl Deref for Hash {
-    type Target = drawbridge_hash::Hash;
+impl Deref for Entry {
+    type Target = Vec<u8>;
```

Comment:
This is not going to work if we intend to validate that e.g. hashes in the signature and the hash of the contents match. We probably want an enum here with `JOSE` signature and plain entry being two variants